### ISSUE TITLE
Error in running make all command patched OQS openssl fork

### ISSUE THREAD

**Issue by NomanNasirMinhas** [wolfSSL/osp#159]
I am following all the instructions as given [here](https://github.com/wolfSSL/osp/tree/master/oqs). I am getting following errors after running the final command i.e, make all. I have attached the screenshots of the history as well as the start of the error in make all command. You can check the output of history command to see if all the steps were followed properly. The OpenSSL version is OpenSSL 3.0.11 19 Sep 2023 (Library: OpenSSL 3.0.11 19 Sep 2023). Also, even though this screenshot is of error on raspberry pi, I faced the same error on Ubuntu as well.

![2](https://github.com/wolfSSL/osp/assets/28532113/c9aaf52d-c378-415d-8482-50c7481e8d18)
![Capture](https://github.com/wolfSSL/osp/assets/28532113/c57a6a66-607a-4baf-a9f1-b48a234afea8)

**Comment by anhu** [comment]
Hello again @NomanNasirMinhas , 

the version of OQS's openSSL seems to be looking algorithms that are no longer supported in your version of liboqs.  Root cause is likely the same as your other issue that you opened, your version of liboqs is too new.  Please try release 0.8.0. 

Warm regards, Anthony

**Comment by NomanNasirMinhas** [comment]
Nope! Still the same issue. I am using 0.8.0 liboqs. You can see the output of `history` command to verify

![Screenshot 2023-12-28 at 7 11 53 PM](https://github.com/wolfSSL/osp/assets/28532113/81c830d5-95e5-4a2d-ba26-4741f23e8c1d)

**Comment by anhu** [comment]
Hello @NomanNasirMinhas , 

It seems I will need to spend some time to update our code.  Perhaps it has gotten stale.  

In the meantime, can you please let me know a bit about your project and any kind of organizations or institutions that are associated with this work?  Please let me know a bit about yourself such as where you are located and whether you are doing this project out of personal, academic or professional interest.  

Warm regards, Anthony

**Comment by anhu** [comment]
> In the meantime, can you please ...

Sorry about that, no need.  You have already done so via different channels.

**Comment by shyamarshid** [comment]
I wanted to share my experience and a solution that worked for me regarding the issue with generating Post-Quantum Cryptography (PQC) certificates using wolfSSL with liboqs. Initially, following the wolfSSL instructions, I used liboqs version 0.8.0 but encountered errors similar to what Nasir mentioned.

I attempted to address this by downgrading wolfSSL to versions 5.5.2 and 5.5.4, but these changes didn't resolve the issue. However, I found a breakthrough in an older version of the README.md file in the wolfSSL osp repository. Specifically, this commit: [5ad16875](https://github.com/wolfSSL/osp/commit/5ad16875bcfd2d8240914c117f79b5e18e136497#diff-f69459a91ee1278e2fd5ffcd2a4e3d27299d163fdb254460e4f0688ddc2189fa), which referenced a specific commit in the liboqs repository. This commit corresponds to liboqs version 0.7.2-dev.

Using this specific version of liboqs (0.7.2-dev), along with the patch provided in the same commit (link to the patch: [here](https://github.com/wolfSSL/osp/commit/5ad16875bcfd2d8240914c117f79b5e18e136497#diff-06eba2d2b7751cffaa3eaaeedf1d9cca9fd5dfd6fa2ff1e22d2b1cd6a2dca3d9)), I successfully generated PQC certificates, including Dilithium, using the OpenSSL fork of wolfSSL.

It seems that, at least in my case, the specific combination of wolfSSL with liboqs version 0.7.2-dev and the mentioned patch was necessary to get things working correctly.

I hope this information is helpful for others facing similar issues!
